pkg/query/dns/dns.go

```diff
@@ -68,7 +77,7 @@ func TimedReverseLookup(ips []string, timeout time.Duration) (ipToDomain map[str
 			if LookupResult.Success {
 				ipToDomain[LookupResult.IP] = LookupResult.Domain
 			}
-		case <-time.After(timeout):
+		case <-queryCtx.Done(): // timeout case
 			pending = 0
```

Comment:
Nitpick: Why not just `return` here? using the `pending` variable as abort criterion feels a bit non-idiomatic...

Reply:
Not sure. Not my code :P. Allow me to refactor once I move ahead with the augmentation plugins upon printing.